nix/bitstream.nix

```
@@ -0,0 +1,29 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+{
+  pkgs,
+  lrPkgs,
+  pythonEnv,
+  FLAKE_GIT_COMMIT,
+  FLAKE_GIT_DIRTY,
+}: {
+  build = pkgs.writeShellApplication {
+    name = "build";
```

Could these have slightly more useful names e.g.

```suggestion
    name = "sonata-bitstream-build";
```

---

Done